pyproject.toml

```diff
@@ -26,6 +27,37 @@ exclude_lines = [
     "if TYPE_CHECKING:",
 ]
 
+[tool.coverage.coverage_conditional_plugin.rules]
```

Comment:
How many of these are we actually using? Given that this is going to be a bit noisy, my inclination is only to include the one's we're actually using. There's a couple of very specific Python versions that are interesting - mostly for "does importlib have the API we need" reasons, rather than for Python version reasons.